ee/api/test/test_conversation.py

```diff
@@ -162,19 +162,32 @@ def test_unauthenticated_request(self):
         )
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
 
-    def test_streaming_error_handling(self):
-        def raise_error():
-            yield "some content"
-            raise Exception("Streaming error")
+    def test_streaming_error_after_content_yields_proper_message(self):
```

Comment:
I see some failing tests, but they don't seem related to your changes, so I re-ran the jobs to be sure.